specification/common-types/resource-management/v5/networksecurityperimeter.json

```diff
@@ -0,0 +1,337 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-09-15-preview",
+    "title": "Common types for network security perimeters based on a common API specification"
+  },
+  "definitions": {
+    "PublicNetworkAccess": {
+      "type": "string",
+      "description": "Allow, disallow, or let Network Security Perimeter configuration to evaluate public network access to Storage Account. Value is optional but if passed in, must be 'Enabled', 'Disabled' or 'SecuredByPerimeter'.",
+      "enum": [
+        "Enabled",
+        "Disabled",
+        "SecuredByPerimeter"
+      ],
+      "x-ms-enum": {
+        "name": "PublicNetworkAccess",
+        "modelAsString": true
+      }
```

Comment:
It would be nice to include "values" in the "x-ms-enum" with a description for each value -- these descriptions will then be shown in the generated REST API Docs. See the [autorest documentation][] for the syntax of the "values" field in "x-ms-enum".

There are other `x-ms-enums` in this file that would also benefit from added descriptions on the values.

[autorest documentation]: https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-enum